habs_portal/static/css/plotting.css

```diff
@@ -0,0 +1,28 @@
+#page-content-wrapper {
+  padding-top: 15px;
+}
+
+.container-fluid {
+  height: 100%;
+}
+
+.full-height {
+  height: 100%;
+}
+
+.plotting-row1 {
+  height: 450px;
+  padding-bottom: 20px;
+}
+
+.plotting-row2 {
+  height: calc(100% - 300px); // not sure why this # works
+}
+
+#timeseries-container {
+  height: 370px;
```

Comment:
@lukecampbell It's late and I didn't feel like banging my head against the wall to try and figure out how to get the dummy image to size properly.  So since I think we are going to go the fixed height route for SVG's, this is where you would get/put that magic #.